src/test/java/core/basesyntax/service/TransactionParseServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionParseServiceImplTest {
+    private TransactionParseService parserService;
+    private List<String> dailyTransactionList;
+
+    @Before
+    public void setUp() throws Exception {
+        parserService = new TransactionParseServiceImpl();
+        dailyTransactionList = new ArrayList<>();
+    }
+
+    @Test
+    public void parseList_Ok() {
+        dailyTransactionList.add("type,fruit,quantity");
+        dailyTransactionList.add("b,banana,20");
+        dailyTransactionList.add("b,apple,100");
+        List<FruitTransaction> lists = parserService.parse(dailyTransactionList);
+        assertEquals(2,lists.size());
+        assertEquals("banana",lists.get(0).getFruit());
```

Comment:
Is it enough to know the size of the list and the name of the first fruit to make sure the parsing is correct?